airflow/dag_processing/bundles/base.py

```diff
@@ -84,7 +87,9 @@ def path(self) -> Path:
         """
         Path for this bundle.
 
-        Airflow will use this path to load/execute the DAGs from the bundle.
+        Airflow will use this path to find/load/execute the DAGs from the bundle.
+        You can change this path during refreshes, but it cannot change between refreshes.
```

Comment:
> You can change this path during refreshes, but it cannot change between refreshes.

I do not understand what this means

Reply:
```suggestion
        The path can change when `refresh` is called, but it cannot change between refreshes.
```

Better?

Reply:
We had a chat and are going to leave this off.